mongodb/mongodb-atlas-cli
# CLOUDP-124942: Introduce a new meta-package called mongodb-atlas

## Proposed changes

Introduce a new meta-package called mongodb-atlas



_Jira ticket:_ CLOUDP-124942



Closes #[issue number]

## Checklist



- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)

## Further comments


Steps to test this:
1. Created a Dockerfile like this
```dockerfile
FROM ubuntu:20.04

RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates wget gnupg
RUN wget -qO - https://www.mongodb.org/static/pgp/server-5.0.asc | apt-key add -
RUN echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-5.0.list && apt-get update
RUN wget -qO atlas.deb https://mongodb-mongocli-build.s3.amazonaws.com/mongocli-master/dist/05719234991a7b02db215797ac8805dca69fb826_22_06_08_11_05_05/mongodb-atlas_1.1.2-next_linux_x86_64.deb
RUN apt-get install -y ./atlas.deb

ENTRYPOINT echo "*** running 'mongosh --version' ***" && mongosh --version && echo "*** running 'atlas --version' ***" && atlas --version
```
2. ran `docker build -t atlas-test .`
3. ran `docker run --rm -it atlas-test`
at this point you would enter the shell of the container and see a message similar to this:
```
*** running 'mongosh --version' ***
1.5.0
*** running 'atlas --version' ***
atlascli version: 1.1.2
git version: 5fedea9a26c492345a9d8317218ee3aff97b179c
Go version: go1.18.2
   os: linux
   arch: amd64
   compiler: gc
```
